contracts/token/ERC20/ERC20.sol

```diff
@@ -125,7 +125,7 @@ contract ERC20 is IERC20 {
      * @param value The amount to be transferred.
      */
     function _transfer(address from, address to, uint256 value) internal {
-        require(to != address(0));
+        require(to != address(0), "empty or zero transaction recipient");
```

Comment:
I'd change this to `transfer` instead of `transaction`, which may cause the reader to think we're talking about the Ethereum transaction.

I'd also go with 'zero' as opposed to 'empty' when talking about the zero address.

Reply:
sorry, my mistake. It should be transfer. I will change is to "zero transfer recipient".

Reply:
I changed it to "transfer recipient is zero". I think it's more meaningful. Please suggest if something I need to change.